### PR TITLE
Footer , pagination , some bug fixed  

### PR DESCRIPTION
**Pull request by AbusiddikAntor**
footer problem fixed and pagination add

### CODE REVIEW

File: frontend/src/pages/Admin/Report.jsx [hunk 1]
```diff
@@ -372,15 +372,16 @@ const Report = () => {
...
+                      {
+                      [...reports?.docs]
+                        // ?.sort((a, b) =>
+                        //   a.username.toLowerCase() > b.username.toLowerCase()
+                        //     ? 1
+                        //     : a.username.toLowerCase() <
+                        //       b.username.toLowerCase()
+                        //     ? -1
+                        //     : 0
+                        // )
```
Comment (devBasar) [inline frontend/src/pages/Admin/Report.jsx:384]: please remove coment

File: frontend/src/pages/employee/EditEmployee.jsx [hunk 2]
```diff
@@ -14,13 +14,14 @@ import { useUploadSingleMutation } from "../../redux/baseAPI.js";
...
-      /^[a-zA-Z][a-zA-Z0-9\s]*$/,
-      "Address must start with a character and can include characters and numbers"
-    ),
+  // address: yup
+  //   .string()
+  //   .required("Address is required")
+  //   .matches(
+  //     /^[a-zA-Z][a-zA-Z0-9\s]*$/,
+  //     "Address must start with a character and can include characters and numbers"
+  //   ),
```
Comment (devBasar) [inline frontend/src/pages/employee/EditEmployee.jsx:23]: remove it

**Review by devBasar — COMMENTED** [review]
need to remove some comment

File: frontend/src/pages/employee/EditEmployee.jsx (hunk 2 above)
Comment (AbusiddikAntor) [reply frontend/src/pages/employee/EditEmployee.jsx:23]: remove comment tnx

Comment (devBasar) [reply frontend/src/pages/employee/EditEmployee.jsx]: you are welcome

Comment (AbusiddikAntor) [reply frontend/src/pages/employee/EditEmployee.jsx]: remove comment tnx

**Review by devBasar — APPROVED** [review]
comment removed

**Pull request merged**